python_gdeploy/wrapper/gdeploy_wrapper.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+import uuid
+
+GDEPLOY_CONFIG_PATH = "/var/run/python_gdeploy/config_"
+
+
+def add_section(section):
+    return "[" + section + "]\n"
+
+
+def cook_gdeploy_config(recipe):
```

Comment:
What is the schema of the config? INI style? Please use appropriate config parsers (yaml or ini or json)